deriv_api/cache.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    from deriv_api.deriv_api import DerivAPI
```

Comment:
```suggestion
    from deriv_api import DerivAPI
```
As stated before this could be rewritten like this, is shorter but require rewriting the other parts.